core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -175,14 +175,42 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
   .oppia-exploration-summary-tile .avatars-num-minus-one {
     color: white;
     font-family: Roboto;
-    font-size: x-small;
+    font-size: xx-small;
     font-weight: bold;
-    left: 18px;
+    left: 12px;
     position: absolute;
-    top: 18px;
-    padding-left: 4px;
+    top: 17px;
   }
 
+  @-moz-document url-prefix() { 
+  .oppia-exploration-summary-tile .avatars-num-minus-one {
+    color: white;
+    font-family: Roboto;
+    font-size: xx-small;
+    font-weight: bold;
+    left: 12px;
+    position: absolute;
+    top: 17px;
+    padding-left: 0px;
+    padding-right: 1px;
+  }
+}
+
+  .oppia-exploration-summary-tile .contributor-indicator {
+  position: absolute;
```

Comment:
Please alphabetize the CSS attributes within the rule (i.e. border-radius first, then bottom, etc.